renderdoc/driver/metal/metal_types.cpp

```diff
@@ -612,4 +672,85 @@ RenderPassDescriptor::operator MTL::RenderPassDescriptor *()
   return objc;
 }
 
+ComputePassSampleBufferAttachmentDescriptor::ComputePassSampleBufferAttachmentDescriptor(
+    MTL::ComputePassSampleBufferAttachmentDescriptor *objc)
+    : startOfEncoderSampleIndex(objc->endOfEncoderSampleIndex()),
```

Comment:
This looks like a typo and should be 

    : startOfEncoderSampleIndex(objc-> startOfEncoderSampleIndex()),